flow/network.h

```diff
@@ -646,4 +647,39 @@ class INetworkConnections {
 	// Returns the interface that should be used to make and accept socket connections
 };
 
+struct DiskFailureInjector : FastAllocated<DiskFailureInjector> {
+	static DiskFailureInjector* injector() {
+		auto res = g_network->global(INetwork::enFailureInjector);
```

Comment:
Why not just use a static local?

Reply:
`g_network` globals are per virtual process in simulation.